### PR TITLE
Require Dart 2.19, latest lints, skip hanging test

### PR DESCRIPTION
**Pull request by kevmoo**
...instead of ignoring it
Also some tiny spelling fixes

### CODE REVIEW

**Comment by devoncarew** [comment]
Can we hold off on this until I land my two outstanding PRs? I do plan to come back to those, and would like to avoid merge issues if possible :)

**Comment by parlough** [comment]
I spent multiple minutes googling "skin hanging test", thinking I was uninitiated to some unique test type. Once I read the changes I finally understood... 🤣

**Review by devoncarew — APPROVED** [review]

**Pull request merged**
